tools/replay/lib/rp_helpers.py

```diff
@@ -69,25 +70,24 @@ def plot_lead(rs, lid_overlay):
 
 
 def maybe_update_radar_points(lt, lid_overlay):
-  ar_pts = []
-  if lt is not None:
+  if lt is not None and hasattr(lt, 'trackList'):
```

Comment:
this should just be handled in the caller. not sure why it was written like this with the "maybe"

Reply:
I agree it's quite strange.

Reply:
It's now handled in the caller and it passed all tests.